addon/authenticators/oidc.js

```diff
@@ -196,6 +199,11 @@ export default BaseAuthenticator.extend({
 
     this._scheduleRefresh(expireTime, refresh_token);
 
-    return { access_token, refresh_token, userinfo, id_token, expireTime };
+    this._currentToken= { access_token, refresh_token, userinfo, id_token, expireTime };
```

Comment:
`_currentToken` would suggest that this property contains a token string. However, this is the parsed data of the current token. I think this should be named something like `_currentTokenData`